app/views/books/_translations.html.erb

```diff
@@ -1,4 +1,8 @@
 <p>
+This book is available in
+  <a href="/book/en">English</a>.
+</p>
+<p>
```

Comment:
Yeah, I was thinking about it but I wasn't too sure how to formulate that (the latter).

Originally I thought having something like "Pro Git is written in English" but it sounded a bit weird to me.

What do you think about:

  * This book is available in English
  * Full translation available in **
  * Partial translations available in **